poollet/volumepoollet/vcm/generic.go

```diff
@@ -143,28 +144,28 @@ func (g *Generic) Start(ctx context.Context) error {
 	return nil
 }
 
-func (g *Generic) GetVolumeClassFor(ctx context.Context, name string, caps *ori.VolumeClassCapabilities) (*ori.VolumeClass, error) {
+func (g *Generic) GetVolumeClassFor(ctx context.Context, name string, caps *ori.VolumeClassCapabilities) (*ori.VolumeClass, int64, error) {
```

Comment:
Do we really want to have an `int64` return value or use some `Quantity` type here?